tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_replay_cache.c

```diff
@@ -204,9 +388,11 @@ static void test_rx_power_replay_attack(void)
 static const struct bst_test_instance test_rpc[] = {
 	TEST_CASE(tx, immediate_replay_attack, "RPC: perform replay attack immediately"),
 	TEST_CASE(tx, power_replay_attack,     "RPC: perform replay attack after power cycle"),
+	TEST_CASE(tx, rpl_frag, "RPC: Send messages after double IVI Update"),
 
 	TEST_CASE(rx, immediate_replay_attack, "RPC: device under immediate attack"),
 	TEST_CASE(rx, power_replay_attack,     "RPC: device under power cycle reply attack"),
+	TEST_CASE(rx, rpl_frag, "RPC: Test RPL fragmentation after double IVI Update"),
```

Comment:
Probably it is worth to add a test to check that the replay cache still works after the power cycle to catch potential desynchronization between entries and related flags during setting storage.